File: solutions/Previous/221205/BracketRotationTest.java
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class BracketRotationTest {
    @Test
    void solutionCase1() {
        BracketRotation test = new BracketRotation();

        String brackets = "[](){}";
        assertThat(test.solution(brackets)).isEqualTo(3);
    }

    @Test
    void solutionCase2() {
        BracketRotation test = new BracketRotation();

        String brackets = "[)(]";
        assertThat(test.solution(brackets)).isEqualTo(0);
    }
}

File: solutions/Previous/221206/bracketRotation.test.js
import solution from './bracketRotation';

describe('bracketRotation', () => {
  it('solution', () => {
    expect(solution('[](){}')).toBe(3);
    expect(solution('}]()[{')).toBe(2);
    expect(solution('[)(]')).toBe(0);
    expect(solution('}}}')).toBe(0);
  });
});
